config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -178,7 +178,7 @@ periodics:
       - --image-family=ubuntu-2004-lts
       - --image-project=ubuntu-os-cloud
       - --provider=gce
-      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
+      - --test_args=--ginkgo.skip=\[Driver:.gcepd\]|\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\] --minStartupPods=8
```

Comment:
can you please add a comment explaining why it's skipped above like here: https://github.com/kubernetes/test-infra/blob/9635aedc198c947e261f3b5d8660160895fd1abb/config/jobs/kubernetes/sig-node/containerd.yaml#L1127